person.rb

```diff
@@ -1,6 +1,6 @@
-require_relative 'name_able'
-require_relative 'trimmer_decorator '
-require_relative 'capital_decorator'
+require './name_able'
+require './trimmer_decorator '
```

Comment:
- [ ] I had the same error as the last code reviewer 👇🏽 

```ruby
franclobo@DESKTOP-E09US2D:~/microverse/oop-school-lib$ ruby main.rb
<internal:/home/franclobo/.rbenv/versions/3.2.2/lib/ruby/site_ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require': cannot load such file -- ./trimmer_decorator  (LoadError)
        from <internal:/home/franclobo/.rbenv/versions/3.2.2/lib/ruby/site_ruby/3.2.0/rubygems/core_ext/kernel_require.rb>:85:in `require'
        from /home/franclobo/microverse/oop-school-lib/person.rb:2:in `<top (required)>'
        from /home/franclobo/microverse/oop-school-lib/student.rb:1:in `require_relative'
        from /home/franclobo/microverse/oop-school-lib/student.rb:1:in `<top (required)>'
        from /home/franclobo/microverse/oop-school-lib/app.rb:1:in `require_relative'
        from /home/franclobo/microverse/oop-school-lib/app.rb:1:in `<top (required)>'
        from main.rb:1:in `require_relative'
        from main.rb:1:in `<main>'
```

- [ ] Please 🙏🏽 correct the typo error, there is an extra space at the end of `require './trimmer_decorator '` to be able to use the app without issues. After that, everything is working.

Reply:
ok thank you
done